samples/bookinfo/routing/route-rule-reviews-90-10.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: reviews
+spec:
+  hosts:
+    - reviews
+  http:
+  - route:
+    - destination:
+        host: reviews
+        subset: v1
+      weight: 90
+    - destination:
+        host: reviews
+        subset: v2
+      weight: 10
```

Comment:
where are the other files?

Reply:
this is a new file, all other files are in samples/bookinfo/routing, they are not changed